exercises/concept/plane-tickets/plane_tickets.py

```diff
@@ -0,0 +1,32 @@
+""" Plane Tickets Exercise """
```

Comment:
PEP-257 doesn't should any space padded docstrings. The summary line is also supposed to be a complete sentence and end in a period.